docker/docker-compose.yml

```diff
@@ -28,67 +32,132 @@ services:
   rabbitmq:
     image: rabbitmq:3.9-management
     container_name: rabbitmq
-    environment:
-      - RABBITMQ_DEFAULT_USER=${RABBITMQ_DEFAULT_USER}
-      - RABBITMQ_DEFAULT_PASS=${RABBITMQ_DEFAULT_PASS}
-    volumes:
-      - amqpdata:/var/lib/rabbitmq
-    restart: unless-stopped
-  
-  opencti:
-    image: docker.darklight.ai/opencti
-    build: ../opencti-platform
-    container_name: opencti
-    environment:
-      - NODE_OPTIONS=--max-old-space-size=8096
-      - APP__PORT=8080
-      - APP__ADMIN__EMAIL=${OPENCTI_ADMIN_EMAIL}
-      - APP__ADMIN__PASSWORD=${OPENCTI_ADMIN_PASSWORD}
-      - APP__ADMIN__TOKEN=${OPENCTI_ADMIN_TOKEN}
-      - APP__APP_LOGS__LOGS_LEVEL=error
-      - REDIS__HOSTNAME=redis
-      - REDIS__PORT=6379
-      - ELASTICSEARCH__URL=http://elasticsearch:9200
-      - MINIO__ENDPOINT=minio
-      - MINIO__PORT=9000
-      - MINIO__USE_SSL=false
-      - MINIO__ACCESS_KEY=${MINIO_ROOT_USER}
-      - MINIO__SECRET_KEY=${MINIO_ROOT_PASSWORD}
-      - RABBITMQ__HOSTNAME=rabbitmq
-      - RABBITMQ__PORT=5672
-      - RABBITMQ__PORT_MANAGEMENT=15672
-      - RABBITMQ__MANAGEMENT_SSL=false
-      - RABBITMQ__USERNAME=${RABBITMQ_DEFAULT_USER}
-      - RABBITMQ__PASSWORD=${RABBITMQ_DEFAULT_PASS}
-      - SMTP__HOSTNAME=${SMTP_HOSTNAME}
-      - SMTP__PORT=25
-      - PROVIDERS__LOCAL__STRATEGY=LocalStrategy
-      - SUBSCRIPTION_SCHEDULER__ENABLED=false
-      # - PROVIDERS__OPENID__STRATEGY=OpenIDConnectStrategy 
-      # - "PROVIDERS__OPENID__CONFIG__LABEL=Login with OpenID"
-      # - PROVIDERS__OPENID__CONFIG__ISSUER=${OPENID_ISSUER}
-      # - PROVIDERS__OPENID__CONFIG__CLIENT_ID=${OPENID_CLIENT_ID}
-      # - PROVIDERS__OPENID__CONFIG__CLIENT_SECRET=${OPENID_CLIENT_SECRET}
-      # - "PROVIDERS__OPENID__CONFIG__REDIRECT_URIS=[\"https://demo.opencti.io/auth/oic/callback\"]"
     ports:
-      - "8080:8080"
-    depends_on:
-      - redis
-      - elasticsearch
-      - rabbitmq
+      - "5672:5672"
     restart: unless-stopped
   
-  graphql:
```

Comment:
Did you intend to take our the GraphQL configuration for the Cyio GraphQL server?  I believe that RJ likely put that in to support our booting both OpenCTI and Cyio's GraphQL servers.

Reply:
Thus was not intentional. I reverted the docker-compose to the develop version minus minio.